src/isomorphic/classic/types/__tests__/ReactPropTypesProduction-test.js

```diff
@@ -23,7 +23,8 @@ describe('ReactPropTypesProduction', function() {
     __DEV__ = false;
     oldProcess = process;
     global.process = {
-      env: Object.assign({}, process.env, {NODE_ENV: 'production'}),
+      cwd: process.cwd,
+      env: Object.assign({}, process.env, {NODE_ENV: JSON.stringify('production')}),
```

Comment:
yeah, good catch. I think I saw some glitch when not to stringify, but it seems not - updated.